### DISCUSSION TITLE
Possible to run the connectors on different machines?

### DISCUSSION THREAD

**Discussion by RebeccaZirbs** [eclipse-edc/Samples#308] [Q&A] [answered]
Hello, is it possible to run the connectors from the samples repository on different machines and enable simple data exchange between them? I would greatly appreciate any help!

Thank you in advance.

**Comment by paullatzelsperger** [comment, marked as answer]
What are you asking? if it's possible to do networking between two computers? 
I'm sorry if this comes across as condescending, but the question really sounds almost too basic. Most dataspaces _don't_ run on one single computer, so EDC (like almost all other software these days) is engineered towards distributed/networked computing. And as long as you know your counter-party's hostname or IP address you're good to go.

> I would greatly appreciate any help!

Not sure what I can say other than: just replace any `localhost` with the appropriate IP addresses/hostnames
